DqtNotifications/terraform/app.tf

```diff
@@ -0,0 +1,153 @@
+locals {
```

Comment:
Can we move the terraform folder to the root (initially it was thought we might have multiple projects in the repo but that's not going to be the case)